crates/holochain/src/conductor/conductor.rs

```diff
@@ -1186,6 +1188,121 @@ pub async fn integration_dump(vault: &EnvRead) -> ConductorApiResult<Integration
         .await
 }
 
+/// Dump the full integration json state.
+/// Careful! This will return a lot of data.
+pub async fn full_integration_dump(
+    vault: &EnvRead,
+    dht_ops_cursor: Option<u64>,
+) -> ConductorApiResult<FullIntegrationStateDump> {
+    vault
+        .async_reader(move |txn| {
+            let integrated = query_dht_ops_from_statement(
+                &txn,
+                "
```

Comment:
i thought we were making our sql into separate files now, ping @neonphog would that be the case here?

Reply:
Yeah, I'd say we should opportunistically move SQL to the standalone files so we get the benefit of formatting, linting, and syntax highlighting in editors. We've got lots of other inline SQL, so not dire if you have more pressing priorities.

Reply:
Where should I move this to? To `holochain_state`, create a new module for dht_ops and create a `get_dht_ops` function?

Reply:
I'd just add it to https://github.com/holochain/holochain/blob/develop/crates/holochain_sqlite/src/sql.rs make it `pub` and use it directly instead of the literal string here.